File: 2.파스칼의 삼각형/solution(기섭).c
#include <stdio.h>
 
int main(){
    int k,n,arr[31][31];
    scanf("%d %d", &n, &k);
    for(int i=1;i<=n;i++){
        arr[i][1] = arr[i][i] = 1;
        for(int j=2; j<i;j++)
            arr[i][j]=arr[i-1][j]+arr[i-1][j-1];
    }
    printf("%d",arr[n][k]);
}

File: 2.파스칼의 삼각형/solution(기섭).py
n,k=input().split()
arr = [[0]*31 for i in range(31) ]
n=int(n)
k=int(k)
for i in range (1,n+1):
    arr[i][i] = arr[i][1] = 1
    for j in range (2,i):
        arr[i][j]=arr[i-1][j]+arr[i-1][j-1]
print(arr[n][k])